src/main/java/com/revature/services/UserServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.revature.services;
+
+import com.revature.models.User;
+import com.revature.repositories.UserRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService
+{
+    @Autowired
+    UserRepo ur;
+
+    @Autowired
+    BCryptPasswordEncoder bCryptEncoder;
+
+    public User addUser(User user)
+    {
+        user.setPassword(bCryptEncoder.encode(user.getPassword()));
+        return ur.save(user);
+    }
+
+    @Override
+    public User getUserById(int id) {
```

Comment:
We probably want to return an Optional here instead of returning an empty object. Makes checking for it easier in the controller.